src/components/DatePicker.vue

```diff
@@ -462,6 +462,13 @@
       getMonth(date) {
         return this.i18n["month-names"][fecha.format(date, 'M') - 1] + (this.showYear ? fecha.format(date, ' YYYY') : '');
       },
+      getMonthDiff(d1, d2) {
+        let d1Y = d1.getFullYear();
+        let d2Y = d2.getFullYear();
+        let d1M = d1.getMonth();
+        let d2M = d2.getMonth();
+         return (d2M+12*d2Y)-(d1M+12*d1Y);
```

Comment:
please give your variables meaningful names. We should probably prevent this method from returning negative numbers as well